src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/templates/fields/Email.tsx

```diff
@@ -1,11 +1,15 @@
 import type {FieldProps} from '@givewp/forms/propTypes';
 
-export default function Email({label, placeholder, fieldError, inputProps}: FieldProps) {
+export default function Email({label, requiredLabel, placeholder, fieldError, inputProps}: FieldProps) {
     return (
         <label>
-            <span>{label}</span>
-            <input type="email" aria-invalid={fieldError ? 'true' : 'false'}
-                   placeholder={placeholder} {...inputProps} />
+            <span>{requiredLabel}</span>
```

Comment:
Where does the original `label` prop go now?  I see this change to the other field components as well.  Here it's leftover but not applied.

Reply:
I cleaned this, and some other stuff, up in c58f88f.